MyLibrary/MyLibraryApp.Application/Models/Books/BookResponseModel.cs

```diff
@@ -0,0 +1,38 @@
+﻿using MyLibraryApp.DataAccess.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyLibraryApp.Application.Models.Books
+{
+    public class BookResponseModel
+    {
+        public int Id { get; set; }
+        public string Title { get; set; }
+        public string Author { get; set; }
+        public string Details { get; set; }
+        public int? CategoryId { get; set; }
+        public int? PublishingHouseId { get; set; }
+        public int? PublishingYear { get; set; }
+        public byte[]? Picture { get; set; }
```

Comment:
This data type should be changed to string and store the URL to a picture (we discusses this, just mentioning it).